core/domain/stats_jobs_continuous.py

```diff
@@ -411,27 +412,72 @@ def entity_classes_to_map_over(cls):
 
     @staticmethod
     def map(item):
```

Comment:
Could handle later, but -- this job is going to take forever if it iterates through all versions each time, right -- even old ones? I wonder if we could get it not to do that -- e.g. if an exploration is at version N, then once any stat for version N-1 is computed, it need not ever be recomputed.